src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -87,7 +87,8 @@ internal abstract partial class CommonCompiler
             TextWriter consoleOutput,
             TouchedFileLogger touchedFilesLogger,
             ErrorLogger errorLoggerOpt,
-            ImmutableArray<AnalyzerConfigOptionsResult> analyzerConfigOptions);
+            ImmutableArray<AnalyzerConfigOptionsResult> analyzerConfigOptions,
+            AnalyzerConfigOptionsResult globalConfigOptions);
```

Comment:
nit: since global options now take precedence, consider moving them up in the parameter list (before `analyzerConfigOptions`).

Reply:
Global options are lower precedence than analyzer options.

---
In reply to: [471708791](https://github.com/dotnet/roslyn/pull/45871#discussion_r471708791) [](ancestors = 471708791)